src/OrbitGl/GraphTrack.cpp

```diff
@@ -31,19 +35,18 @@ GraphTrack<Dimension>::GraphTrack(CaptureViewElement* parent, TimeGraph* time_gr
 
 template <size_t Dimension>
 float GraphTrack<Dimension>::GetHeight() const {
-  float scale_factor = IsCollapsed() ? 1 : Dimension;
-  float height = layout_->GetTrackTabHeight() + GetLegendHeight() +
-                 layout_->GetTextBoxHeight() * scale_factor +
-                 layout_->GetSpaceBetweenTracksAndThread() + layout_->GetEventTrackHeightFromTid() +
-                 layout_->GetTrackBottomMargin();
+  float height = layout_->GetTrackTabHeight() + layout_->GetTrackContentTopMargin() * 2 +
```

Comment:
It's not clear for me why you are counting TrackContentTopMargin twice. Consider to add a comment.